src/ApiService/ApiService/FeatureFlags.cs

```diff
@@ -8,4 +8,5 @@ public static class FeatureFlagConstants {
     public const string EnableBlobRetentionPolicy = "EnableBlobRetentionPolicy";
     public const string EnableDryRunBlobRetention = "EnableDryRunBlobRetention";
     public const string EnableWorkItemCreation = "EnableWorkItemCreation";
+    public const string EnableContainerRetentionPolicies = "EnableContainerRetentionPolicies";
```

Comment:
You need to add it here as well: https://github.com/microsoft/onefuzz/blob/main/src/deployment/bicep-templates/feature-flags.bicep

Reply:
Done, defaulted to enabled.